src/deploy-cromwell-on-azure/Deployer.cs

```diff
@@ -44,9 +44,21 @@ public class Deployer
         private const string InputsContainerName = "inputs";
 
         private readonly CancellationTokenSource cts = new CancellationTokenSource();
+
+        private readonly List<string> requiredResourceProviders = new List<string>
+            {
+                "Microsoft.Compute",
+                "Microsoft.Network",
+                "Microsoft.Batch",
+                "Microsoft.DocumentDB",
+                "Microsoft.insights",
+                "Microsoft.Storage"
```

Comment:
Is Microsoft.Billing also required? https://docs.microsoft.com/en-us/azure/role-based-access-control/resource-provider-operations#microsoftbilling

Reply:
No, the system does not access any Microsoft.Billing operations.